packages/react-table/src/components/Table/examples/Table.md

```diff
@@ -365,6 +369,159 @@ class SortableTable extends React.Component {
 }
 ```
 
+### Sortable with toolbar
```

Comment:
I think it may be more useful to name this example something like `Sortable - Responsive` so that at the top level it's clear that THIS is the responsive solution for sortable tables.